libcontainer/configs/validate/validator.go

```diff
@@ -151,6 +151,16 @@ func (v *ConfigValidator) sysctl(config *configs.Config) error {
 				return fmt.Errorf("sysctl %q is not allowed in the hosts network namespace", s)
 			}
 		}
+		if config.Namespaces.Contains(configs.NEWUTS) {
+			switch s {
+			case "kernel.domainname":
+				// This is namespaced and there's no explicit OCI field for it.
+				continue
```

Comment:
i dont think we need to explicitly single out `kernel.domainname` here since its not a TODO for OCI either right ?

Reply:
This code is **allowing** `kernel.domainname` which is why it needs to be singled out (the comment explains why it's allowed and `kernel.hostname` isn't). If you try to use `kernel.domainname` without this patch it will fail because `runc` thinks its not namespaced and fails container creation.